go/vt/vtgate/vtgate.go

```diff
@@ -565,7 +563,7 @@ func (vtg *VTGate) VStream(ctx context.Context, tabletType topodatapb.TabletType
 
 // GetGatewayCacheStatus returns a displayable version of the Gateway cache.
 func (vtg *VTGate) GetGatewayCacheStatus() TabletCacheStatusList {
-	return vtg.resolver.GetGatewayCacheStatus()
+	return vtg.gw.CacheStatus()
```

Comment:
The indirection was necessary when we had both DiscoveryGateway and TabletGateway. Now that we have only one Gateway, we can go to that directly instead of going through resolver.